docs/site/translation/getting-started/guide.md

```diff
@@ -293,10 +298,14 @@ To change it, you have to flag the item for committee review:
 1. Click on the “**Flag for Review button**”.
 2. On the new page, you'll see a message box.
 3. Enter the change that you want to make, and add a justification for changing it.
-4. Then click **“Post**”.
-5. Towards the end of data collection cycle, the Technical Committee will review the change request and either accept it, or reject it with comments.
-
-Note: The **Flag for Review** button will be available only when the item has a 20 vote value AND there is a suggestion in the **Others** column.
+    - Make sure that the justification is clear to people who don't speak your language.
```

Comment:
Can we consider adding a line recommending that vetters provide links to reference materials?